components/retro-go/targets/t-deck-plus/config.h

```diff
@@ -0,0 +1,115 @@
+// Target definition
+#define RG_TARGET_NAME             "T-DECK-PLUS"
+
+// Storage
+#define RG_STORAGE_ROOT             "/sd"
+#define RG_STORAGE_SDSPI_HOST       SPI2_HOST
+#define RG_STORAGE_SDSPI_SPEED      SDMMC_FREQ_DEFAULT
+
+// Audio
+#define RG_AUDIO_USE_INT_DAC        0   // 0 = Disable, 1 = GPIO25, 2 = GPIO26, 3 = Both
+#define RG_AUDIO_USE_EXT_DAC        1   // 0 = Disable, 1 = Enable
+
+// Board-specific
+#define T_DECK_BOARD_POWER          GPIO_NUM_10
+#define T_DECK_RADIO_CS             GPIO_NUM_9
+#define T_DECK_KBD_ADDRESS          0x55
+#define T_DECK_KBD_MODE_RAW_CMD     0x03
+
+#define RG_CUSTOM_PLATFORM_INIT()                             \
+    gpio_set_direction(T_DECK_BOARD_POWER, GPIO_MODE_OUTPUT); \
+    gpio_set_level(T_DECK_BOARD_POWER, 1);                    \
+    gpio_set_direction(RG_GPIO_SDSPI_CS, GPIO_MODE_OUTPUT);   \
+    gpio_set_direction(T_DECK_RADIO_CS, GPIO_MODE_OUTPUT);    \
+    gpio_set_direction(RG_GPIO_LCD_CS, GPIO_MODE_OUTPUT);     \
+    gpio_set_level(RG_GPIO_SDSPI_CS, 1);                      \
+    gpio_set_level(T_DECK_RADIO_CS, 1);                       \
+    gpio_set_level(RG_GPIO_LCD_CS, 1);                        \
+    gpio_set_pull_mode(RG_GPIO_SDSPI_MISO, GPIO_PULLUP_ONLY); \
+    rg_task_delay(50);
+
+// Video
+#define RG_SCREEN_DRIVER            0   // 0 = ILI9341
+#define RG_SCREEN_HOST              SPI2_HOST
+#define RG_SCREEN_SPEED             SPI_MASTER_FREQ_80M
+#define RG_SCREEN_BACKLIGHT         1
+#define RG_SCREEN_WIDTH             320
+#define RG_SCREEN_HEIGHT            240
+#define RG_SCREEN_ROTATE            0
+#define RG_SCREEN_MARGIN_TOP        0
+#define RG_SCREEN_MARGIN_BOTTOM     0
+#define RG_SCREEN_MARGIN_LEFT       0
+#define RG_SCREEN_MARGIN_RIGHT      0
```

Comment:
I've replaced margins with something else, can you replace those lines with:

```c
#define RG_SCREEN_VISIBLE_AREA      {0, 0, 0, 0}
#define RG_SCREEN_SAFE_AREA         {0, 0, 0, 0}
```

(It's just for consistency with the other target.h)